src/components/context-menu/ContextCircleButton.js

```diff
@@ -27,6 +29,42 @@ const ContextIcon = styled(Text).attrs(({ theme: { colors } }) => ({
 });
 
 export default function ContextCircleButton(props) {
+  if (android) {
```

Comment:
I think it's better to define two separate components and export them using Platform.select, since this android condition basically defines a separate component.